modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -22,7 +22,7 @@ The supported profiles are:
 
 * Lite: This installs only Tekton Pipelines.
 * Basic: This installs Tekton Pipelines, Tekton Triggers, and Tekton Chains.
-* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components, including Tekton Pipelines, Tekton Triggers, Tekton Chains, {pac}, and Tekton Addons. Tekton Addons includes the `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart`, and `ConsoleYAMLSample` resources, as well as the tasks available using the cluster resolver from the `openshift-pipelines` namespace.
+* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components, including Tekton Pipelines, Tekton Triggers, Tekton Chains, {pac}, and Tekton Addons. Tekton Addons includes the `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart`, and `ConsoleYAMLSample` resources, as well as the tasks available using the cluster resolver from the `openshift-pipelines` namespace.
```

Comment:
>as well as the tasks available using the cluster resolver from the `openshift-pipelines` namespace.

Reply:
> @savitaashture @aneeshmbhat shouldn't we mention ecosystem tasks and step actions here?

I think `as well as the tasks available using the cluster resolver from the `openshift-pipelines` namespace.` will explain that instead of saying ecosystem task :thinking: